grafana-plugin/src/models/filters/filters.ts

```diff
@@ -17,6 +17,9 @@ export class FiltersStore extends BaseStore {
   @observable.shallow
   public values: { [page: string]: FiltersValues } = {};
 
+  @observable
+  public isLoading = true;
```

Comment:
easier to track now if the filters are loading or not

Reply:
so don't see the reason of having  `isLoading` flag